docs/articles/vs-test-adapter/Adapter-Release-Notes.md

```diff
@@ -254,7 +254,7 @@ Exception thrown while loading tests if In-Proc VSTest DataCollector is used (Th
 
 ### Features
 
-* [461](https://github.com/nunit/nunit3-vs-adapter/issues/461)  Publish symbols with the adapter for debugging into the adapter and nunit
+* [461](https://github.com/nunit/nunit3-vs-adapter/issues/461)  Publish symbols with the adapter for debugging into the adapter and NUnit
```

Comment:
Maybe "the adapter and the NUnit framework"? I think "NUnit" means all the components. The NUnit Framework is a component with the assembly name `nunit`.

Reply:
I don't mind this suggestion in general, but this PR is only to address capitalization, not phrasing. 👍